content/reliability/docs/architecting-for-static-stability.md

```diff
@@ -0,0 +1,19 @@
+
+Second is the assumption that the correlated outage will not impact control plane orchestration systems. While these systems are designed to be highly available, the number of systems involved in a failover scenario spans multiple AWS services, Kubernetes core components, and user installed Kubernetes operators. Evacuation results in a large spike in traffic across these systems, and correlated impact on any of these systems can cause an evacuation strategy to fail.
+
+Third is the introduction of bi-modal system behavior. Availability Zone evacuation is a rare event and introduces a new operational mode in addition to the standard operational mode of the system. This mechanism is expensive to test, and thus, rarely tested. Changes to the system over time may violate original assumptions and test results, the impact of which won’t surface until it’s too late.
+
+Static stability solves this problem and is a recommended best practice in both AWS and Kubernetes. Consider spreading application replicas across multiple failure domains, including across virtual machines and Availability Zones. In the event of a correlated outage, the automation halts changes, allowing the existing capacity to handle the workload’s traffic. Networking layers shift traffic away from the impacted Availability Zone towards the replicas in healthy Availability Zones. Application scaling should correspond to its reliability requirements. To survive a single Availability Zone outage, replicas in other AZs must be preemptively scaled to handle the additional load. Workload performance can be nonlinear, so it’s essential to test. To survive a multiple availability zone event, larger overhead is required. At a fundamental level, customers must choose the right balance of redundancy and cost that meets their use case.
+
+For Kubernetes applications, this is best achieved using [pod anti-affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) and [topology spread constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints). Consider using anti-affinity for the key `kubernetes.io/hostname` and topology spread constraints for the key `topology.kubernetes.io/zone` in Deployments. Together, these will ensure that the Kubernetes scheduler spreads the Deployment’s Pods across multiple Availability Zones and nodes. The workload should be both vertically and horizontally sized, using resource requests and replica count, respectively. [AWS Fault Injection Service](https://docs.aws.amazon.com/fis/latest/userguide/fis-actions-reference.html#eks-actions-reference) can help with testing these configurations against a simulated availability event and validate utilization assumptions when traffic shifts away from the impacted zone.
```

Comment:
Some of this guidance is already in the guides under reliability/applications. Can we consolidate? Why does this need to be under a separate heading (architecting for static stability)?

Reply:
I think this PR possibly predates that guide.

Reply:
@realvz can you see where this fits? I don't think it should be a standalone page. It overlaps with content on other pages.